### PR TITLE
[Bug-23160] Correct code example in revXMLEvaluateXPath

### PR DESCRIPTION
**Pull request by livecodealice**
Changed the expression used in the bottom code example so that it will give the correct result instead of a crash.

### CODE REVIEW

File: docs/dictionary/function/revXMLEvaluateXPath.lcdoc [hunk 1]
```diff
@@ -63,7 +63,7 @@ instance, given xml data of
     
 then
 
-    put "/bookstore/book/[price&lt;40]/" into pXPathExpression
+    put "/bookstore/book[price&lt;40]/" into pXPathExpression
     put revXMLEvaluateXPath(pDocID, pXPathExpression)
```
Comment (livecodepanos) [inline docs/dictionary/function/revXMLEvaluateXPath.lcdoc:67]: @livecodesam there is another instance of the same typo/mistake that needs to be corrected

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 2c4b3cd

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 2c4b3cdfed99c82125444c259ec85ed7bb2342fc

**Comment by livecode-vulcan** [comment]
😎 test success 2c4b3cdfed99c82125444c259ec85ed7bb2342fc

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/981)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/389)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/388)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/387)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/388)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2506)
* try-community-universal-ios-iphoneos14.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos14.4/builds/35)
* try-community-universal-ios-iphoneos14.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos14.1/builds/84)
* try-community-universal-ios-iphonesimulator14.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator14.4/builds/35)
* try-community-universal-ios-iphonesimulator14.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator14.1/builds/82)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2533)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2522)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2520)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2557)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2493)

**Pull request merged**